dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -160,28 +160,43 @@ def test_true_categorical_report(self):
         self.assertTrue(report["categorical"])
         six.assertCountEqual(
             self,
-            ['unique_count', 'unique_ratio', 'categories'], report['statistics']
+            ['unique_count', 'unique_ratio', 'categories', 'categorical_count'], report['statistics']
         )
         self.assertEqual(3, report["statistics"]["unique_count"])
         self.assertEqual(0.25, report["statistics"]["unique_ratio"])
         self.assertCountEqual(
             ["a", "b", "c"], report["statistics"]["categories"]
         )
+        expected_dict = {"a": 3, "b": 4, "c": 5}
+        self.assertTrue(report["statistics"]["categories"])
+        self.assertEqual(expected_dict,
+                             report["statistics"]["categorical_count"])
 
     def test_false_categorical_report(self):
         df_non_categorical = pd.Series(list(map(str, range(0, 20))))
         profile = CategoricalColumn(df_non_categorical.name)
         profile.update(df_non_categorical)
 
         report = profile.profile
+        report.pop("times")
         six.assertCountEqual(
-            self, ['categorical', 'statistics', 'times'], report)
+            self, ['categorical', 'statistics'], report)
         self.assertFalse(report["categorical"])
         six.assertCountEqual(
             self, ['unique_count', 'unique_ratio'], report['statistics']
         )
         self.assertEqual(20, report["statistics"]["unique_count"])
         self.assertEqual(1.0, report["statistics"]["unique_ratio"])
+        self.assertFalse("categorical_count" in report["statistics"])
+
+        expected_profile = dict(
+            categorical=False,
+            statistics=dict([
+                ('unique_count', 20),
+                ('unique_ratio', 1),
+            ]),
+        )
+        self.assertEqual(report, expected_profile)
```

Comment:
Since we have this profile we can remove the following asserts since they become redundant:
```python
six.assertCountEqual(self, ['categorical', 'statistics'], report)
self.assertFalse(report["categorical"])
six.assertCountEqual(self, ['unique_count', 'unique_ratio'], report['statistics'])
self.assertEqual(20, report["statistics"]["unique_count"])
self.assertEqual(1.0, report["statistics"]["unique_ratio"])
```